src/test/java/com/uid2/shared/store/RotatingServiceLinkStoreTest.java

```diff
@@ -62,10 +62,10 @@ public void loadContent_emptyArray_loadsZeroServiceLinks() throws Exception {
     @Test
     public void loadContent_multipleServiceLinksStored_loadsAllServiceLinks() throws Exception {
         JsonArray content = new JsonArray();
-        ServiceLink l1 = addServiceLink(content, "abc123", 1, 123, "Test Service 1", Set.of());
-        ServiceLink l2 = addServiceLink(content, "abc123", 2, 123, "test1", Set.of(Role.MAPPER));
-        ServiceLink l3 = addServiceLink(content, "ghi789", 1, 123, "Test Service 1", Set.of(Role.MAPPER, Role.SHARER));
-        ServiceLink l4 = addServiceLink(content, "jkl1011", 3, 124, "test2", null);
+        ServiceLink l1 = addServiceLink(content, "abc123", 1, 123, "Test Service 1", Set.of(), false);
+        ServiceLink l2 = addServiceLink(content, "abc123", 2, 123, "test1", Set.of(Role.MAPPER), true);
```

Comment:
we are introducing a mix of enabled/disabled in the test but what tests or assertions are we doing with these?

Reply:
Where will the tests be that assert that disabling a service link or service, actually disables something?